package.json

```diff
@@ -50,7 +50,8 @@
     "doc:typeddocfull": "npx del-cli doc/full && typedoc --options ./typedoc.full.json",
     "doc:typeddocstrip": "npx del-cli doc/strip && typedoc --options ./typedoc.stripInternal.json",
     "apitest:build": "webpack --mode development --config ./apitest/webpack.config.js",
-    "apitest:serve": "webpack serve --mode development --config ./apitest/webpack.config.js"
+    "apitest:serve": "webpack serve --mode development --config ./apitest/webpack.config.js",
+    "prepare": "npm run build"
```

Comment:
why is this change in here?
It seems unrelated to the actual PR

Reply:
Yes it is, sorry... I needed it to reference the branch and keep working. If this isn't there I can't work with something like this: https://github.com/guillemcordoba/golden-layout-custom-elements/blob/714660975ef54059ee314b0c1f030210c2a873f2/package.json#L22.

The prepare script is run after the npm install is done targeting github, which compiles golden-layout and creates the dist folder.

Do you want me to remove it?

Reply:
Would be good to remove it, I think it breaks when installed in a production environment, because the devdeps of golden-layout are missing there.

Reply:
Okey then :) No problem. BTW if you have seen that [golden-layout-custom-elements](https://github.com/guillemcordoba/golden-layout-custom-elements) I'm starting, I'd be happy to contribute that in any way you find appropriate, if it's useful to others (for now this is for myself).